library/std/src/sync/lazy_lock.rs

```diff
@@ -20,6 +20,8 @@ union Data<T, F> {
 /// A value which is initialized on the first access.
 ///
 /// This type is a thread-safe [`LazyCell`], and can be used in statics.
+/// Therefore, any dereferencing call will block the calling thread if
```

Comment:
Nit: I would either drop the `Therefore` or reword the sentence to something like `Since initialization may be called from multiple threads, any dereferencing...`. Since it's not just the fact that it can be used in statics that mean there may be multiple accessors, you could just as easily have this in an allocation.

Reply:
Fare point; I see how it can be misinterpreted to be "statics" related. I like your suggestion. Will update it soon.

Thanks for all the helpful feedback, @tgross35. We'll wait for the lib-team crew to move forward, then. Cheers!